framework/src/org/checkerframework/framework/type/AnnotatedTypeFactory.java

```diff
@@ -2955,7 +2955,21 @@ private AnnotationMirror getDeclAnnotation(
 
         Set<AnnotationMirror> results = AnnotationUtils.createAnnotationSet();
         // Retrieving the annotations from the element.
-        results.addAll(elements.getAllAnnotationMirrors(elt));
+        List<? extends AnnotationMirror> fromEle = elements.getAllAnnotationMirrors(elt);
+        for (AnnotationMirror annotation : fromEle) {
+            try {
```

Comment:
It happens when calling areSame on an annotation mirror whose class isn't on the classpath.  So I guess I can move the try catch there.